changelogs/unreleased/bump-libcurl-to-7.83.0.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
+
+* Bump Curl from 7.80.0 to 7.83.0 for static build (gh-6029).
```

Comment:
The same comments as in #7119 are applicable here.

Reply:
Updated.